geode-apis-compatible-with-redis/src/main/java/org/apache/geode/redis/internal/executor/sortedset/ZRemRangeByLexExecutor.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional information regarding
+ * copyright ownership. The ASF licenses this file to You under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package org.apache.geode.redis.internal.executor.sortedset;
+
+
+import java.util.List;
+
+import org.apache.geode.redis.internal.data.RedisKey;
+import org.apache.geode.redis.internal.executor.RedisResponse;
+
+public class ZRemRangeByLexExecutor
+    extends AbstractSortedSetRangeExecutor<SortedSetLexRangeOptions> {
+  @Override
+  public boolean isRev() {
+    return false;
+  }
+
+  @Override
+  public SortedSetLexRangeOptions createRangeOptions(List<byte[]> commandElements) {
+    return new SortedSetLexRangeOptions(commandElements, isRev());
+  }
+
+  @Override
+  public RedisResponse getEmptyResponse() {
+    return RedisResponse.emptyArray();
```

Comment:
The empty response for this class should be `RedisResponse.integer(0);` since a no-op range remove returns 0 rather than an empty array.